### DISCUSSION TITLE
Pin Table Headers Option

### DISCUSSION THREAD

**Discussion by Tommy-Ruse** [GitbookIO/community#591] [Feature Requests]
Add an option to pin Table Headers with sticky behavior so that they remain visible when scrolling long columns

**Comment by muchisx** [comment]
Sticky table headers would be very helpful! +1

**Comment by aikoii** [comment]
Hiya 👋! You can now freeze table headers 🎉
<img width="2010" height="1002" alt="CleanShot 2026-04-02 at 18 17 59@2x" src="https://github.com/user-attachments/assets/8d00dd3d-aa44-421e-b97d-31aa01f0506c" />
